ansible/tasks/logging/install_fluentd.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe
+  win_get_url:
+    url: "{{ ruby_fetch_url }}"
+    dest: "{{ tempdir_win.path }}\\ruby_devkit.exe"
+- name: Run ruby devkit exe
+  win_package:
+    path: "{{ tempdir_win.path }}\\ruby_devkit.exe"
+    creates_path: "C:\\Ruby26-x64\\bin\\ruby.exe"
```

Comment:
`rubyinstaller-devkit-2.4.7-1-x86.exe` implies it is a 32-bit binary. But the path you are creating implies 64-bit. Why?

Reply:
I missed the change here, I will take care of it. However, this path would be version dependent, should we want to change the version of devkit in `main.yml`, so I will make the path here a variable.
I am not creating a path specifically - this is a requirement for win_package module so it won't re-install if already installed. More information found [here](https://docs.ansible.com/ansible/latest/modules/win_package_module.html#parameters)

Reply:
What is the default behaviour here? If we don't use `creates_path`, would the installation happen at `C:\<product_name>`? If yes, I am in favour of that

Reply:
@ravisantoshgudimetla `creates_path` does not control the installation directory. That can be specified as a separate argument in the `arguments` section of `win_package` module.

`creates_path` has the definition of `Will check the existence of the path specified and use the result to determine whether the package is already installed.` in the official documentation, however its execution is counter-intuitive. If the path mentioned in `creates_path` is valid, the task will be skipped. So we need to be very specific about the path mentioned in 'creates_path`.
Coming back to the discussion of installation directory, if we mention installation directory as, say -`D:\\ruby`, the installation will happen there. However we cannot give `D:\\ruby` as the path in `creates_path` because this  path can (and will) continue to exist if ruby is uninstalled from the system. What the `win_package` module ideally expects here is the absolute application path of the exe when it is run from the system so it knows with absolute certainty whether the said exe is already installed on the system

Reply:
I have chosen to scrape the `win_package` module since it depends on pining the `creates_path` value, and instead tested the code with `win_shell` module, with enough conditionals to ensure safe execution.
I believe it is a better option because our code can then be agnostic of Ruby version as well as the installation directory